src/ProjectTemplates/BlazorTemplates.Tests/playwrightSettings.ci.win.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Will this fallback to whatever defaults Playwright wants in a Windows scenario or will it read from the `playwrightSettings.json`? Curious about config precedence here.

Reply:
Yep, last config setting applied wins, and we only use some files based on whether or not they meet a condition.